src/operator/tensor/ordering_op-inl.h

```diff
@@ -378,16 +378,19 @@ void TopKImpl(const RunContext &ctx,
     << "The total element_num is " << element_num << ", but the selected IDType can only represent "
     << mxnet::common::MaxIntegerValue<IDType>() << " elements";
   Tensor<xpu, 3, DType> dat = src.FlatTo3D<xpu, DType>(axis, axis, s);
-  size_t temp_size = 0;
+  index_t temp_size = 0;
```

Comment:
Maybe `size_t` is more appropriate here and the arguments have to be casted instead